src/pricing-fetcher.ts

```
@@ -91,35 +97,42 @@ export class PricingFetcher implements Disposable {
-			this.cachedPricing = pricing;
-			logger.info(`Loaded pricing for ${pricing.size} models`);
-			return pricing;
-		}
-		catch (error) {
-			return this.handleFallbackToCachedPricing(error);
+				this.cachedPricing = pricing;
+				logger.info(`Loaded pricing for ${pricing.size} models`);
+				return pricing;
+			},
+			catch: error => error,
+		});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)